### PR TITLE
Remove workaround for MiniMQTT (Issue #25)

### PR DESCRIPTION
**Pull request by brentru**
Removes library workaround patched in https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/pull/33 in favor of using hostname without `https:` appended to the broker string

### CODE REVIEW

**Comment by brentru** [comment]
@jimbobbennett  I can't tag you as a reviewer. However, could you please review this? Thanks!

**Comment by tannewt** [comment]
I invited @jimbobbennett to collaborate on the repo so they can review officially and merge.

**Comment by brentru** [comment]
Thanks, Scott!

File: adafruit_azureiot/iot_mqtt.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def device_twin_desired_updated(self, desired_property_name: str, desired_proper
         :param int desired_version: The version of the desired property that was updated
         """
 
-    def device_twin_reported_updated(self, reported_property_name: str, reported_property_value, reported_version: int) -> None:
+    def device_twin_reported_updated(self, reported_property_name: str, reported_property_value, reported_version: int,) -> None:
```
Comment (jimbobbennett) [inline adafruit_azureiot/iot_mqtt.py:94]: Does this extra comma do anything?

**Comment by jimbobbennett** [comment]
Noticed a few extra commas - not sure if they are needed. I'll do a full check and test later this week, sorry tied up the Microsoft Build at the moment.

File: adafruit_azureiot/iot_mqtt.py (hunk 1 above)
Comment (brentru) [reply adafruit_azureiot/iot_mqtt.py:94]: @jimbobbennett  Hrm, I didn't add them. These may have been added by `black`'ing the file.

**Comment by brentru** [comment]
@jimbobbennett  No rush, enjoy BUILD

**Comment by askpatrickw** [comment]
>Black will add trailing commas to expressions that are split by comma where each element is on its own line. This includes function signatures.

https://black.readthedocs.io/en/stable/the_black_code_style.html#trailing-commas

**Comment by jimbobbennett** [comment]
@brentru - I've updated the PR with a couple of changes:

* Removed the rogue commas to make it easier to see the changes
* Removed the `_try_create_mqtt_client` method and moved the code into the `_create_mqtt_client` method to avoid having a method that does nothing except call another method
* Removed a comment referencing the bug in miniMQTT

I've tested it on a PyPortal using a number of the examples. Can you review please?

**Pull request merged**